project/accounts/api.py

```diff
@@ -18,12 +18,13 @@
 from accounts.permissions import IsProfileOwnerOrDuringRegistrationOrReadOnly
 from accounts.serializers import ProfileSerializer, ProfileListSerializer
 from accounts.forms import UpdateProfileImage
-from api.utils import get_account
-from api.models import Thread, Civi, Activity
+from threads.utils import get_account, json_response

```

Comment:
This should fix the error mentioned by @gorkemarslan 

```suggestion
from api.utils import get_account
from threads.utils import json_response
```